### PR TITLE
add params to the aq_locations function

### PR DESCRIPTION
**Pull request by RobinKohrs**
This is far away from being done. But maybe it has some kind of sense:) I tried to add the four parameters to the `aq_locations`-function and then tried to define their behavior in `utils.R`.

### CODE REVIEW

File: R/locations.R [hunk 1]
```diff
@@ -12,6 +12,10 @@
 #' @template radius
 #' @template limit
 #' @template page
+#' @template siteType
```
Comment (maelle) [inline R/locations.R:15]: I wonder whether we should make all param names "snake case" although the API params are "camel case":

File: R/locations.R [hunk 2]
```diff
@@ -12,6 +12,10 @@
...
+#' @template activationDate
+#' @template completeness
```
Comment (maelle) [inline R/locations.R:17]: there should be two params in the docs as in the function (completeness_from and completeness_to)

Comment (maelle) [reply R/locations.R:17]: after that `devtools::document()` should be run.

File: man/aq_locations.Rd [hunk 3]
```diff
@@ -15,6 +15,7 @@ aq_locations(
   longitude = NULL,
   radius = NULL,
   page = NULL
+  siteType = NULL
```
Comment (maelle) [inline man/aq_locations.Rd:18]: There's a missing comma (but `devtools::document()` needs to be run anyway)

File: R/locations.R [hunk 4]
```diff
@@ -68,7 +74,16 @@ aq_locations <- function(country = NULL, city = NULL, location = NULL,# nolint
...
+                         page = page,
+                         siteType = siteType,
+                         completeness_from = completeness_from,
+                         completeness_to = completeness_to,
+                         activation_date_from = activation_date_from,
+                         activation_date_to = activation_date_to,
+                         inletHeight_from = inletHeight_from,
+                         inletHeight_to = inletHeight_to)
+
+  return(argsList)
```
Comment (maelle) [inline R/locations.R:86]:
```suggestion
```

You forgot to remove this :wink:

**Review by maelle — COMMENTED** [review]
Thanks a lot! I added a few comments. Locally if I remove the line `return(argsList)`, tests pass again.

File: R/locations.R (hunk 4 above)
Comment (RobinKohrs) [reply R/locations.R:86]: oh yes of course 🙈

File: R/locations.R (hunk 2 above)
Comment (RobinKohrs) [reply R/locations.R:17]: Oh, yes.. But this is true for activationDate and inletHeight as well right?

File: R/locations.R (hunk 1 above)
Comment (RobinKohrs) [reply R/locations.R:15]: hm good question. I really like the "camel case" version, but I don't have enough experience at all to judge about this. I think however that `activation_date_from` should better be `activationDate_from` and `activationDate_to` right?

File: man/aq_locations.Rd (hunk 3 above)
Comment (RobinKohrs) [reply man/aq_locations.Rd:18]: oh of course! But I'm really sorry. I think I'm  a little confused with this and should maybe read more about the `devtools`-package before diving deeper into this.

File: R/locations.R (hunk 4 above)
Comment (RobinKohrs) [reply R/locations.R:86]: Sorry for reaching out again... I didn't have any time to look into this. Probably this is solved by now anyways.

Comment (maelle) [reply R/locations.R:86]: actually no :joy: so if you have time to work on this feel free to ping me. 
I need to do a release of ropenaq next week, but this feature could go to the release after that.

**Pull request closed without merging**